src/main/kotlin/com/terraformation/backend/accelerator/api/DeliverablesController.kt

```diff
@@ -0,0 +1,207 @@
+            ),
+        ))
+  }
+
+  @ApiResponse200
+  @ApiResponse404
+  @GetMapping("/{deliverableId}")
+  @Operation(
+      summary = "Gets the details of a single deliverable and its submission documents, if any.")
+  fun getDeliverable(@PathVariable deliverableId: DeliverableId): GetDeliverableResponsePayload {
+    return GetDeliverableResponsePayload(
+        DeliverablePayload(
+            category = DeliverableCategory.LegalEligibility,
+            descriptionHtml = "<p>A description</p>",
+            documents =
+                listOf(
+                    SubmissionDocumentPayload(
+                        Instant.now(),
+                        "Project's articles of incorporation",
+                        DocumentStore.Google,
+                        SubmissionDocumentId(13974),
+                        "Incorporation Documents_2024-02-28_Omega_Projects articles of incorporation.doc",
+                    )),
+            feedback = "Is this a joke? This is a bunch of cat photos, not a legal document.",
+            id = deliverableId,
+            internalComment = "These guys are real jokers.",
+            name = "Incorporation Documents",
+            organizationId = OrganizationId(1),
+            organizationName = "Test Org",
+            participantId = ParticipantId(2),
+            participantName = "Random Participant",
+            projectId = ProjectId(3),
+            projectName = "Omega Project",
+            status = SubmissionStatus.Rejected,
+            type = DeliverableType.Document,
+        ))
+  }
+
+  @ApiResponse(
+      responseCode = "307",
+      description =
+          "If the current user has permission to view the document, redirects to the document " +
+              "on the document store. Depending on the document store, the redirect URL may or " +
+              "may not be valid for only a limited time.")
+  @ApiResponse404
+  @GetMapping("/{deliverableId}/documents/{documentId}")
+  @Operation(summary = "Gets a single submission document from a deliverable.")
+  fun getDeliverableDocument(
+      @PathVariable deliverableId: DeliverableId,
+      @PathVariable documentId: SubmissionDocumentId
+  ): RedirectView {
+    val url = URI("https://dropbox.com/")
+
+    val redirectView = RedirectView(url.toString(), false, false)
+    redirectView.setStatusCode(HttpStatus.TEMPORARY_REDIRECT)
+
+    return redirectView
+  }
+}
+
+data class ListDeliverablesElement(
+    val category: DeliverableCategory,
+    @Schema(description = "Optional description of the deliverable in HTML form.")
+    val descriptionHtml: String?,
+    val id: DeliverableId,
+    val name: String,
+    @Schema(
+        description =
+            "Number of documents submitted for this deliverable. Only valid for deliverables of " +
+                "type Document.")
+    val numDocuments: Int?,
+    val organizationId: OrganizationId,
+    val organizationName: String,
```

Comment:
I included both IDs and names so this won't be dependent on the client having already fetched stuff like the list of projects and so that it'll work in the admin view where the IDs can point to organizations the user isn't a member of.

The downside is that the responses will be bigger; open to other suggestions if that seems like the wrong tradeoff.

Reply:
I like this approach, simplifies client side code and fewer requests. Client has a way to fetch project name from id if needed (Nick set something up), but we don't have something similar for organizations.
Guessing the response payload sizes should be ok, we aren't dealing with large number of deliverables (yet?).

Reply:
I agree, this is nice.